lib/helpers.js

```diff
@@ -19,4 +22,17 @@ module.exports = {
             .request(method, url, request, options);
     },
 
+    readLaunchesFromFile() {
+        return fs.readFileSync(tempFileForLaunchIds, 'utf8', function(err, data) {
+            if (err) throw err;
+            return data;
+        }).toString().split('|');
+    },
+
+    saveLaunchIdToFile(launch_id) {
```

Comment:
can we make the file in the same format as we have in jvm-based clients?
https://github.com/reportportal/client-java/blob/master/src/main/java/com/epam/reportportal/utils/LaunchFile.java#L41

Reply:
I've added the same behavior for JS Client.